app/templates/organen/orgaan/mandatarissen.hbs

```diff
@@ -5,6 +5,9 @@
       @selectedPeriod={{@model.selectedPeriod}}
       @onSelect={{this.selectPeriod}}
     />
+    {{#unless (await this.model.bestuursOrgaan.isDecretaal)}}
+      <BestuursperiodeNewModal @bestuursorgaan={{this.model.bestuursOrgaan}} />
```

Comment:
this looks like it always shows the modal?

Reply:
I get it! Your component is actually the BestuursperiodeNewButton, that opens a modal on click. Could you rename as such?